src/styles/blocks/container.scss

```diff
@@ -0,0 +1,13 @@
+@import "../utils/variables";
+
+.main__container {
+  width: $container-width;
+  display: flex;
+  justify-content: flex-start;
```

Comment:
I see you've fixed [that](https://github.com/mate-academy/layout_catalog/pull/2037#issuecomment-1300485127), but you forgot to run `deploy` again after the change